### PR TITLE
add 4.6 Operator manifests

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

File: manifests/4.6/ptp-operator.v4.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (fedepaol) [inline manifests/4.6/ptp-operator.v4.5.0.clusterserviceversion.yaml:1]: Should the filename be v4.6.0.xxxx ?

Comment (zshi-redhat) [reply manifests/4.6/ptp-operator.v4.5.0.clusterserviceversion.yaml]: Yes, it should be. I forgot to change it when copying from 4.5.  Thanks @fedepaol !
Updated.

**Pull request merged**